optuna/pruners/hyperband.py

```diff
@@ -57,19 +57,67 @@ class HyperbandPruner(BasePruner):
         most use cases).　Please see Section 3.6 of the `original paper
         <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_ for the detail.
 
+    Example:
+
+        We minimize an objective function with Hyperband pruning algorithm.
+
+        .. testsetup::
+
+            import numpy as np
+            from sklearn.model_selection import train_test_split
+
+            np.random.seed(seed=0)
+            X = np.random.randn(200).reshape(-1, 1)
+            y = np.where(X[:, 0] < 0.5, 0, 1)
+            X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+            classes = np.unique(y)
+            n_train_iter = 100
```

Comment:
How about moving this line somewhere in `testcode`?
You show the actual values of the other arguments of `HyperbandPruner` (i.e., `min_resource` and `reduction_factor `), and users may try to find the value of `n_train_iter`.